node/api/README.md

```diff
@@ -64,14 +67,171 @@ connections will be queued. The queue has a maximum size and when it is
 reached, any new incoming connections will be rejected with an error code 400.
 This is to prevent the node of being overloaded with WebSocket connections.
 
-### General message types
-All messages have the format:
+### Messages overview
+
+The JSON-RPC protocol supports both synchronous RPC and asynchronous notifications.
+An RPC is indicated by the inclusion of an `'id'` parameter. The Aeternity API
+can be used both synchronously and asynchronously. When an asynchronous method is called, the reply will have `'method': '<RequestMethod>.reply'`.
+
+Implementation support exists for batch processing, however this is not yet tested.
+
+### Examples
+
+#### Example, synchronous RPC
+
+```javascript
+{
+  "jsonrpc": "2.0",
+  "id": -576460752303423475,
```

Comment:
Is the negative id expected?

Reply:
I see a number https://www.jsonrpc.org/specification#request_object

so negative looks alright...